Dockerfile

```diff
@@ -20,14 +20,7 @@ RUN set -xeo pipefail && \
     rm "/root/.wget-hsts" && \
     rm "/tmp/glibc.apk" "/tmp/glibc-bin.apk" && \
     curl https://sh.rustup.rs -sSf | sh -s -- -y && \
```

Comment:
you can remove this line as well. it installs rust as a requirement for gnirehtet

Reply:
Done